core/api/src/main/java/com/alipay/sofa/rpc/bootstrap/package-info.java

```diff
@@ -14,8 +14,8 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.alipay.sofa.rpc.bootstrap;
-
 /**
```

Comment:
@jnan806 It's better to leave one line blank. The format will look better.

Reply:
the blank line was deleted by exec `mvn clean package` which was requested by following.

<img width="949" alt="image" src="https://user-images.githubusercontent.com/43714056/200229369-b8e9350c-f2e0-49f7-9aa8-178de049dbb6.png">

Reply:
OK